kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/dmn/DMNResultKS.java

```diff
@@ -105,13 +106,13 @@ public void setModelName(String modelName) {
     }
 
     
-    public String getDecisionName() {
-        return decisionName;
+    public List<String> getDecisionNames() {
+        return decisionNames;
     }
 
     
-    public void setDecisionName(String decisionName) {
-        this.decisionName = decisionName;
+    public void setDecisionNames(List<String> decisionName) {
```

Comment:
Findbugs revealed a bug here. decisionName should be renamed to decisionNames.

Reply:
@sutaakar indeed. time to check my eyes :) fixed